routes/signuproutes.js

```diff
@@ -0,0 +1,16 @@
+const express = require ('express')
```

Comment:
router file-name is signuproutes. So add only routes which are related to signup functionality